src/additional-hooks/index.ts

```diff
@@ -0,0 +1,17 @@
+import {CustomRendererHooks} from '@diplodoc/markdown-it-custom-renderer';
+
+export const mergeAdditionalHooks = (
+    defaultHooks: CustomRendererHooks,
+    additionalHooks?: CustomRendererHooks,
```

Comment:
i think we should go for the variable amount of hooks as parameters with function merging them into one.

about naming: i think mergeHooks would be clear and concise name.